servicecontrol/servicecontrol-instances/distributed-instances.md

```diff
@@ -89,10 +92,9 @@ AuditsSlave -- ingested by --> Slave
 Master -. connected to .-> Slave
 ```
 
-In this example, two ServiceControl instances are used where one is the master and the other the slave for auditing. Some of the endpoints will forward their audit messages to `audit-master` while others will forward their audit messages to `audit-slave`. All error messages are forwared to the `error` queue. ServiceInsight and ServicePulse are connected to the master instance.
+Setup steps:
 
-1. Install the ServiceControl master instance which ingests messages from the `audit-master` queue according to the [installation guidelines](/servicecontrol/installation.md).
-2. Install the ServiceControl slave instance (on separate infrastructure) which ingests messages from the `audit-slave` queue according to the [installation guidelines](/servicecontrol/installation.md). Each slave instance must have a unique name. The ServiceControl instance name and the port are required to configure the master instance. In this example, the name `Particular.ServiceControl.Slave` and the port `33334` is used. Make sure error queue processing is disabled by specifying `!disable` as the error queue field in the ServiceControl Management Utility, or as the error queue parameter of the [Powershell](/servicecontrol/installation-powershell.md) installation script, or by editing `ServiceControl.exe.config` as shown below:
+1. Install the ServiceControl slave instance (on separate infrastructure) which ingests messages from the `audit-slave` queue according to the [installation guidelines](/servicecontrol/installation.md). Each slave instance must have a unique name. The ServiceControl instance name and the port are required to configure the master instance. In this example, the name `Particular.ServiceControl.Slave` and the port `33334` is used. Make sure error queue processing is disabled by specifying `!disable` as the error queue field in the ServiceControl Management Utility, or as the error queue parameter of the [Powershell](/servicecontrol/installation-powershell.md) installation script, or by editing `ServiceControl.exe.config` as shown below:
```

Comment:
Why invert the order of master and slave installation?

Reply:
Because this removes context switch for the user. Before change it's:
 * Do something with the master,
 * Do something with the slave,
 * Do something with the master,

After the change it's:
 * Do something with the slave
 * Do something with the master